Makefile

```diff
@@ -89,6 +92,6 @@ docker-publish: docker
 	docker push brightsparklabs/appcli:${APP_VERSION}
 	docker push brightsparklabs/appcli:latest
 
-all: format isort lint test
+all: format isort lint test pre-commit
```

Comment:
This won't be necessary if we run `pre-commit install` within the `venv` block, as described above

Reply:
Removed